### ISSUE TITLE
Time out

### ISSUE THREAD

**Issue by Azuuuma** [nedex/QuickNTP#11]
Hello, I often encounter connection timeouts when using QuickNTP for time synchronization in China. Could you please add some NTP servers related to China? Thank you very much. Here are the relevant server addresses:

"cn.pool.ntp.org"
"cn.ntp.org.cn"
"ntp.ntsc.ac.cn"
"edu.ntp.org.cn"
"ntp1.aliyun.com"
"time1.cloud.tencent.com"

**Comment by nedex** [comment]
Hello. If you're facing this problem very often i will try to narrow down the actual servers list to include china.
Also because adding all of these servers will be a problem / can't scroll them all using the current slider.
Adding support for a list of servers from a file in the near future would be the definitive solution.

**Comment by Azuuuma** [comment]
> 

Yes, in China, I often experience difficulty in connecting to servers outside of China, or sometimes I can't connect at all. Looking forward to your future updates. Thank you very much for your efforts

**Comment by nedex** [comment]
I'm planning to add a .ini file containing a list of servers to customize the available ones

**Comment by nedex** [comment]
I just made a commit that allows to edit the servers using an ini file (check the dev branch [README](https://github.com/nedex/QuickNTP/blob/dev/README.md)).
There's a build for those who want to test it:

https://github.com/nedex/QuickNTP/actions/runs/15232383676

I've tried it on firmware 19.0.0 using Atmosphere 1.8.0.

Let me know if you have any suggestions of feedback!

**Comment by oloukou1** [comment]
Works well on my erista device, atmosphere 1.9.1 with firmware 19.0.1. Thanks!
